render/image.go

```diff
@@ -155,17 +154,20 @@ func drawSnakeImage(name string, st snakeImageType, dc *gg.Context, bx int, by i
 	}
 	snakeImg = rotateImage(snakeImg, rot)
 
-	dst := dc.Image().(draw.Image)
-	dstRect := image.Rect(
-		int(boardXToDrawX(dc, bx))+SquareBorderPixels+BoardBorder,
-		int(boardYToDrawY(dc, by))+SquareBorderPixels+BoardBorder,
-		int(boardXToDrawX(dc, bx+1))-SquareBorderPixels+BoardBorder,
-		int(boardYToDrawY(dc, by-1))-SquareBorderPixels+BoardBorder,
-	)
+	// dst := dc.Image().(draw.Image)
+	// dstRect := image.Rect(
+	// 	int(boardXToDrawX(dc, bx))+SquareBorderPixels+BoardBorder,
+	// 	int(boardYToDrawY(dc, by))+SquareBorderPixels+BoardBorder,
+	// 	int(boardXToDrawX(dc, bx+1))-SquareBorderPixels+BoardBorder,
+	// 	int(boardYToDrawY(dc, by-1))-SquareBorderPixels+BoardBorder,
+	// )
 
-	srcImage := &image.Uniform{parseHexColor(hexColor)}
+	// srcImage := &image.Uniform{parseHexColor(hexColor)}
 
-	draw.DrawMask(dst, dstRect, srcImage, image.Point{}, snakeImg, image.Point{}, draw.Over)
+	// draw.DrawMask(dst, dstRect, srcImage, image.Point{}, snakeImg, image.Point{}, draw.Over)
```

Comment:
TODO: replicate this logic in the fallback PNG rendering so that it still gets customised